src/go/k8s/README.md

```diff
@@ -24,34 +27,30 @@ Optionaly to run operator locally:
 
 #### Local installation
 
-First clone the repo
-
-```
-git clone https://github.com/vectorizedio/redpanda.git
-```
-
 Create local Kubernetes cluster using KIND
 
 ```
 export KUBECONFIG=your/path/to/kubeconfig.yaml
 kind create cluster --config kind.yaml
 ```
 
-You can simply deploy the latest version stored in GCR
+You can simply deploy the redpanda operator by running the following command
 
 ```
-make deploy
+kubectl apply -k https://github.com/vectorizedio/redpanda/src/go/k8s/config/default
```

Comment:
I am not sure this is good idea, this does not contain the layers added by kustomize as make deploy does... ultimately this should be make deploy OR helm perhaps?

Reply:
I don't understand what kustomize layer is not added? The `-k` parameter will run full kustomize build.

I agree, this should be changed to `helm install` when we will have this. 

The `make deploy` can be describe in `README-DEV.md`

Reply:
oooh! I see, got it! Is kubectl still bundled with ancient kustomize or is that better?

Reply:
It's enough for now. I'm not sure where to find what is exact version of `kustomize` backed in `kubectl`

Reply:
yeah they still use kustomize 2.X.X :( https://github.com/kubernetes/kubectl/issues/818 I think because of that pretty much nobody uses kustomize in kubectl seriously.

Reply:
I follow the rabbit hole and soon we will have [v3.9.2](https://github.com/kubernetes-sigs/kustomize/issues/1500#issuecomment-768454468) :) If we wait a little longer the [v4.0.1](https://github.com/kubernetes/kubernetes/pull/98946).

Reply:
yeah I saw those PRs... well I am fine with keeping this in readme, but I would strongly advice not using this for any kind of production use. Let's just install kustomize and use kustomize explicitly, I don't think it's such a big of a deal...

Reply:
Ok, so I will leave that to the vectorized.io docs @bmansheim :)